sci-mathematics/calc/calc-2.12.5.3.recipe

```diff
@@ -0,0 +1,155 @@
+SUMMARY="C-style arbitrary precision calculator"
+DESCRIPTION="Calc is an interactive calculator which provides for easy large \
+numeric calculations, but which also can be easily programmed for difficult or \
+long calculations. It can accept a command line argument, in which case it \
+executes that single command and exits. Otherwise, it enters interactive mode. \
+In this mode, it accepts commands one at a time, processes them, and displays \
+the answers. In the simplest case, commands are simply expressions which are \
+evaluated."
+HOMEPAGE="http://www.isthe.com/chongo/tech/comp/calc/"
+COPYRIGHT="1999-2016 Landon Curt Noll"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://www.isthe.com/chongo/src/calc/calc-$portVersion.tar.bz2"
+CHECKSUM_SHA256="0fcf60917efb10aab145f1180c278a3ab63d789aad6d5e5f066a4458b5f9b692"
+PATCHES="calc-$portVersion.patchset"
+
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
+
+PROVIDES="
+	calc$secondaryArchSuffix = $portVersion
+	cmd:calc$secondaryArchSuffix = $portVersion
+	cmd:cscript$secondaryArchSuffix = $portVersion
+	lib:libcalc$secondaryArchSuffix
+	lib:libcustcalc$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libreadline$secondaryArchSuffix
+	lib:libhistory$secondaryArchSuffix
+	lib:libncurses$secondaryArchSuffix
+	calc_data == $portVersion
+	"
+
+if [ -z "$secondaryArchSuffix" ]; then
+	SUMMARY_data="Standard resource files for calc"
+	ARCHITECTURES_data="any"
+	PROVIDES_data="
+		calc_data = $portVersion
+		"
+	REQUIRES_data=""
+fi
+
+PROVIDES_devel="
+	calc${secondaryArchSuffix}_devel = $portVersion
+	devel:libcalc$secondaryArchSuffix
+	devel:libcustcalc$secondaryArchSuffix
+	"
+REQUIRES_devel="
+	calc$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libreadline$secondaryArchSuffix
+	devel:libhistory$secondaryArchSuffix
+	devel:libncurses$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc$secondaryArchSuffix
+	cmd:make
+	cmd:awk
+	cmd:sed
+	cmd:diff
+	cmd:egrep
+	cmd:ctags
+	cmd:xargs
+	cmd:cmp
+	cmd:touch
+	cmd:less
+	cmd:strip
+	"
+
+BUILD()
+{
+	make target=Haiku \
+		INCDIR="$includeDir" \
```

Comment:
I think it might be a good idea to replace:
```
	make target=Haiku \
		INCDIR="$includeDir" \
```
by
```
	make \
		target=Haiku \
		INCDIR=$includeDir \
```
or maybe just drop the double quotes around $includeDir, or add double quotes around $libDir and $binDir to have a consistent style.